examples/data-sources/tmc_cluster_group/data-source.tf

```diff
@@ -1,4 +1,4 @@
 # Read Tanzu Mission Control cluster group : fetch cluster group details
-data "tmc_cluster_group" "read_cluster_group" {
+data "tanzu-mission-control_cluster_group" "read_cluster_group" {
   name = "default"
```

Comment:
Similar to data-source.tf, there is a provider.tf. Please update those files as well

Reply:
Done